### DISCUSSION TITLE
SQL Server delimiters: [] or ""?

### DISCUSSION THREAD

**Discussion by tabit01** [dbt-labs/dbt-core#10839] [Q&A]
It seems that the `dbt-fabric` adapter method `adapter.quote()` compiles quoted identifiers to `"column name"`, not `[column name]` as I was expecting. At least to me this was confusing, since the documentation is quite consistent on using bracket delimiters. The quotation marks delimiters also departs from our current naming convention. Is this an issue, or should we just accept and move on?

**Comment by alison985** [comment]
SQL Server is the only dialect I've seen use [] so there is a background cross-dialect / personnel training consideration in the background. Since these are being added programmatically, I'm not sure how much it would be considered to matter until it actually blocks functionality or causes a bug.

Since the adapter is in use already and Microsoft at least knows of its existence, I assume that quotes work fine.

**I recommend moving this issue to the relevant adapter library.** Because the database adapters aren't in dbt-core you'll get better response there.
